include/lsst/meas/extensions/shapeHSM/HsmMomentsControl.h

```diff
@@ -209,9 +219,10 @@ class HsmPsfMomentsAlgorithm : public HsmMomentsAlgorithm {
     typedef std::shared_ptr<const HsmPsfMomentsControl> Control;
 
     /// @brief Initialize with standard field names and customized documentation.
-    HsmPsfMomentsAlgorithm(Control const & ctrl, std::string const & name, afw::table::Schema & schema) :
-        HsmMomentsAlgorithm(name, schema, "Psf adaptive moments algorithm from HSM"), _ctrl(ctrl) {}
-
+    HsmPsfMomentsAlgorithm(Control const & ctrl, std::string const & name, afw::table::Schema & schema,
+                           char const* doc = "PSF adaptive moments algorithm from HSM"  ///< Docstring is exposed for debiased PSF algorithm
```

Comment:
As above, "PSF adaptive moments via the HSM shape algorithm"

Reply:
I can see how this can be confusing as well, if it is interpreted to mean some PSF was adapted to some standard one to measure the moments. I will change this to "Adaptive moments of the PSF via the HSM shape algorithm".